aws-redshiftserverless-namespace/src/main/java/software/amazon/redshiftserverless/namespace/ReadHandler.java

```diff
@@ -56,8 +63,18 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                         .makeServiceCall(this::getNamespaceResourcePolicy)
                         .done((_request, _response, _client, _model, _context) -> {
                             _model.setNamespaceResourcePolicy(Translator.convertStringToJson(_response.resourcePolicy().policy(), logger));
-                            return ProgressEvent.defaultSuccessHandler(_model);
+                            return ProgressEvent.progress(_model, _context);
                         });
+                })
+                .then(progress -> {
+                    return proxy.initiate("AWS-RedshiftServerless-Namespace::SnapshotCopyConfigurations::List", proxyClient, progress.getResourceModel(), callbackContext)
+                            .translateToServiceRequest(Translator::translateToListSnapshotCopyConfigurationsRequest)
+                            .makeServiceCall(this::listSnapshotCopyConfigurations)
+                            .handleError(this::listSnapshotCopyConfigurationErrorHandler)
+                            .done((_request, _response, _client, _model, _context) -> {
+                                _model.setSnapshotCopyConfigurations(Translator.translateToSnapshotCopyConfigurations(_response.snapshotCopyConfigurations()));
+                                return ProgressEvent.defaultSuccessHandler(_model);
```

Comment:
just to make sure, the configuration will be created almost instantly right, it won't be like cluster where we have to wait for a few mins?

Reply:
Yep, it's instant.